app/routes/titiler/algorithms.py

```diff
@@ -26,25 +62,47 @@ def __call__(self, img: ImageData) -> ImageData:
         """Encode Integrated alerts to RGBA."""
         mask = img.array.mask[0].astype(int)
         data = img.data[0]
+        alert_date = data % 10000  # in days since 2014-12-31
+        data_alert_confidence = data // 10000
+
+        r = np.zeros_like(data_alert_confidence, dtype=int)
+        g = np.zeros_like(data_alert_confidence, dtype=int)
+        b = np.zeros_like(data_alert_confidence, dtype=int)
+
+        for _, properties in ALERT_CONFIDENCE_MAP.items():
```

Comment:
minor but kind of messy syntax for iterating. Can you just do like .values()?

Reply:
Of course, good catch!